### DISCUSSION TITLE
Ezengine, Vulkan, OpenAL And True Open Source Nature

### DISCUSSION THREAD

**Discussion by IndieDev99** [ezEngine/ezEngine#1114] [General]
Hello ezengine community and devs, 
I am new to ezengine and I have some questions,
Is vulkan support is ready for production?
Will there be an official OpenAL support?
Is Editor is complete like Unity or Godot? 
Last there is Fmod, DX11 and such things are not free and Open Source. Will there be issue to use Fmod in commercial games without any fee or royalty?

**Comment by na2axl** [comment]
Hey @IndieDev99,

- For the Vulkan support I'm not sure it is ready, but I will let the core team place a better answer for that.
- At the time I don't think there is an official statement in the roadmap that talks about OpenAL. But to integrate a custom audio engine it's an easy task to do, I have built myself an integration for my audio engine [here](https://amplitudeaudiosdk.com)
- From my experience with the engine, the editor is feature complete, and exposes every capability of the engine. And the UI has been improved a lot during the past months. You should be happy using it.
- I do not think you should pay royalties for using DX11 🤔, but for fmod you will have to pay them for a commercial project.

Hope I provided answers to your questions, and welcome to the community!

**Comment by Sanakan8472** [comment]
* @IndieDev99 we are actively working on the Vulkan backend but right now it's experimental and not ready for production.
* For the rest, @na2axl 's answer sums it up nicely.
